includes/level-settings.php

```diff
@@ -32,13 +32,13 @@ function pmprodon_pmpro_membership_level_after_other_settings() {
 	<tr>
 		<th scope="row" valign="top"><label for="donation_min_price"><?php _e( 'Min Amount:', 'pmpro-donations' ); ?></label></th>
 		<td>
-			<?php echo $pmpro_currency_symbol; ?><input type="text" id="donation_min_price" name="donation_min_price" value="<?php echo esc_attr( $min_price ); ?>" />
+			<?php echo $pmpro_currency_symbol; ?><input type="number" id="donation_min_price" name="donation_min_price" value="<?php echo esc_attr( $min_price ); ?>" />

```

Comment:
For these changes, I think we have to stick to text so people can add float values. The preg_replace for these values sanitize it already as it only allows numbers and periods.

Reply:
type="number" allow users to add float values.  also text with the preg replace let users save just dots.
Also mobile keyboards for number fields are great and it's and enhancement in terms of UX and semantically seems more adequate.


<img width="729" alt="image" src="https://github.com/strangerstudios/pmpro-donations/assets/1678457/6917a936-6756-40eb-be95-636a08015b7e">

<img width="920" alt="image" src="https://github.com/strangerstudios/pmpro-donations/assets/1678457/8390a43a-3260-4e39-a544-3bb9448fb3ef">

Reply:
Keeping payment amount fields as `text` fields is consistent with core PMPro:
https://github.com/strangerstudios/paid-memberships-pro/blob/fd6bf1c18315c71013da26d4ab8c0f24e6640591/adminpages/membershiplevels.php#L550

If we want to switch billing amount fields to `number` fields, this should be discussed and implemented in core PMPro first, and then we can start updating Add Ons